drf_kit/models.py

```diff
@@ -126,6 +126,20 @@ class OrderedModelMixin(_OrderedModelBase):
     )
     order_field_name = "order"
 
+    @property
+    def next_in_order(self):
+        try:
+            return self.get_ordering_queryset().get(**{self.order_field_name: self.order + 1})
+        except self.DoesNotExist:
+            return None
+
+    @property
+    def previous_in_order(self):
+        try:
+            return self.get_ordering_queryset().get(**{self.order_field_name: self.order - 1})
+        except self.DoesNotExist:
+            return None
+
```

Comment:
Let it raise the exception instead of returning None, it's closer to some similar actions (eg. [latest](https://docs.djangoproject.com/en/4.0/ref/models/querysets/#latest))